src/auth/src/credentials.rs

```diff
@@ -548,6 +548,7 @@ fn adc_well_known_path() -> Option<String> {
 /// This module is mainly relevant to other `google-cloud-*` crates, but some
 /// external developers (i.e. consumers, not developers of `google-cloud-rust`)
 /// may find it useful.
+#[cfg_attr(test, mutants::skip)]
```

Comment:
Please include a comment explaining why we need to skip this for mutant testing.  Remember to use just `// ` so it does not go into the customer-facing documentation.